### ISSUE TITLE
hung websocket after  websocket server restart

### ISSUE THREAD

**Issue by georgekola** [square/SocketRocket#38]
I  find that  if I restart the websocket server process, the socket rocket client does not detect  the loss of websocket connection i.e. I  do not get a delegate call back on  either didFaileWithError or didCloseWIthCode.

I am using faye (https://github.com/faye/faye-websocket-node) on nodejs as the server and using socket rocket  on ios 5.1 as client and I have implement faye client protocol on top of socket rocket. I find that other clients (faye javascript) is able to detect the server restart fine. 

Basically, the websocket is hung. Trying to send message has no effect. Any help would be greatly appreciated.

**Comment by mikelikespie** [comment]
Doing some housekeeping. Please reopen if still relevant
